Index: requirements.txt
===================================================================
diff --git a/requirements.txt b/requirements.txt
deleted file mode 100644
--- a/requirements.txt	(revision c7759d5fbc8f44ea9cce28f0b92a03f179eb1227)
+++ /dev/null	(revision c7759d5fbc8f44ea9cce28f0b92a03f179eb1227)
@@ -1,5 +0,0 @@
-pandas
-numpy
-matplotlib
-statsmodels
-openpyxl
diff --git a/EPA133a-Lab Assignment 2/README.md b/EPA133a-Lab_Assignment_2/README.md
rename from EPA133a-Lab Assignment 2/README.md
rename to EPA133a-Lab_Assignment_2/README.md
diff --git a/EPA133a-Lab Assignment 2/EPA133a-G07-A2/model/ContinuousSpace/SimpleContinuousModule.py b/EPA133a-Lab_Assignment_2/EPA133a-G07-A2/model/ContinuousSpace/SimpleContinuousModule.py
rename from EPA133a-Lab Assignment 2/EPA133a-G07-A2/model/ContinuousSpace/SimpleContinuousModule.py
rename to EPA133a-Lab_Assignment_2/EPA133a-G07-A2/model/ContinuousSpace/SimpleContinuousModule.py
diff --git a/EPA133a-Lab Assignment 2/requirements.txt b/EPA133a-Lab_Assignment_2/requirements.txt
rename from EPA133a-Lab Assignment 2/requirements.txt
rename to EPA133a-Lab_Assignment_2/requirements.txt
diff --git a/EPA133a-Lab Assignment 2/EPA133a-G07-A2/model/ContinuousSpace/simple_continuous_canvas.js b/EPA133a-Lab_Assignment_2/EPA133a-G07-A2/model/ContinuousSpace/simple_continuous_canvas.js
rename from EPA133a-Lab Assignment 2/EPA133a-G07-A2/model/ContinuousSpace/simple_continuous_canvas.js
rename to EPA133a-Lab_Assignment_2/EPA133a-G07-A2/model/ContinuousSpace/simple_continuous_canvas.js
diff --git a/EPA133a-Lab Assignment 2/.idea/modules.xml b/EPA133a-Lab_Assignment_2/.idea/modules.xml
rename from EPA133a-Lab Assignment 2/.idea/modules.xml
rename to EPA133a-Lab_Assignment_2/.idea/modules.xml
diff --git a/EPA133a-Lab Assignment 2/.idea/.gitignore b/EPA133a-Lab_Assignment_2/.idea/.gitignore
rename from EPA133a-Lab Assignment 2/.idea/.gitignore
rename to EPA133a-Lab_Assignment_2/.idea/.gitignore
diff --git a/EPA133a-Lab Assignment 2/.idea/EPA133a-Lab.iml b/EPA133a-Lab_Assignment_2/.idea/EPA133a-Lab.iml
rename from EPA133a-Lab Assignment 2/.idea/EPA133a-Lab.iml
rename to EPA133a-Lab_Assignment_2/.idea/EPA133a-Lab.iml
diff --git a/EPA133a-Lab Assignment 2/.idea/inspectionProfiles/profiles_settings.xml b/EPA133a-Lab_Assignment_2/.idea/inspectionProfiles/profiles_settings.xml
rename from EPA133a-Lab Assignment 2/.idea/inspectionProfiles/profiles_settings.xml
rename to EPA133a-Lab_Assignment_2/.idea/inspectionProfiles/profiles_settings.xml
diff --git a/EPA133a-Lab Assignment 2/.idea/misc.xml b/EPA133a-Lab_Assignment_2/.idea/misc.xml
rename from EPA133a-Lab Assignment 2/.idea/misc.xml
rename to EPA133a-Lab_Assignment_2/.idea/misc.xml
diff --git a/EPA133a-Lab Assignment 2/EPA133a-G07-A2/README.md b/EPA133a-Lab_Assignment_2/EPA133a-G07-A2/README.md
rename from EPA133a-Lab Assignment 2/EPA133a-G07-A2/README.md
rename to EPA133a-Lab_Assignment_2/EPA133a-G07-A2/README.md
diff --git a/EPA133a-Lab Assignment 2/EPA133a-G07-A2/data/Data Preparation.py b/EPA133a-Lab_Assignment_2/EPA133a-G07-A2/data/Data Preparation.py
rename from EPA133a-Lab Assignment 2/EPA133a-G07-A2/data/Data Preparation.py
rename to EPA133a-Lab_Assignment_2/EPA133a-G07-A2/data/Data Preparation.py
diff --git a/EPA133a-Lab Assignment 2/EPA133a-G07-A2/data/demo-3.csv b/EPA133a-Lab_Assignment_2/EPA133a-G07-A2/data/demo-3.csv
rename from EPA133a-Lab Assignment 2/EPA133a-G07-A2/data/demo-3.csv
rename to EPA133a-Lab_Assignment_2/EPA133a-G07-A2/data/demo-3.csv
diff --git a/EPA133a-Lab Assignment 2/EPA133a-G07-A2/data/demo-2.csv b/EPA133a-Lab_Assignment_2/EPA133a-G07-A2/data/demo-2.csv
rename from EPA133a-Lab Assignment 2/EPA133a-G07-A2/data/demo-2.csv
rename to EPA133a-Lab_Assignment_2/EPA133a-G07-A2/data/demo-2.csv
diff --git a/EPA133a-Lab Assignment 2/EPA133a-G07-A2/data/README.md b/EPA133a-Lab_Assignment_2/EPA133a-G07-A2/data/README.md
rename from EPA133a-Lab Assignment 2/EPA133a-G07-A2/data/README.md
rename to EPA133a-Lab_Assignment_2/EPA133a-G07-A2/data/README.md
diff --git a/EPA133a-Lab Assignment 2/EPA133a-G07-A2/data/demo-1.csv b/EPA133a-Lab_Assignment_2/EPA133a-G07-A2/data/demo-1.csv
rename from EPA133a-Lab Assignment 2/EPA133a-G07-A2/data/demo-1.csv
rename to EPA133a-Lab_Assignment_2/EPA133a-G07-A2/data/demo-1.csv
diff --git a/EPA133a-Lab Assignment 2/EPA133a-G07-A2/model/model.py b/EPA133a-Lab_Assignment_2/EPA133a-G07-A2/model/model.py
rename from EPA133a-Lab Assignment 2/EPA133a-G07-A2/model/model.py
rename to EPA133a-Lab_Assignment_2/EPA133a-G07-A2/model/model.py
diff --git a/EPA133a-Lab Assignment 2/EPA133a-G07-A2/model/scenario.py b/EPA133a-Lab_Assignment_2/EPA133a-G07-A2/model/scenario.py
rename from EPA133a-Lab Assignment 2/EPA133a-G07-A2/model/scenario.py
rename to EPA133a-Lab_Assignment_2/EPA133a-G07-A2/model/scenario.py
diff --git a/EPA133a-Lab Assignment 2/EPA133a-G07-A2/data/_roads3.csv b/EPA133a-Lab_Assignment_2/EPA133a-G07-A2/data/_roads3.csv
rename from EPA133a-Lab Assignment 2/EPA133a-G07-A2/data/_roads3.csv
rename to EPA133a-Lab_Assignment_2/EPA133a-G07-A2/data/_roads3.csv
diff --git a/EPA133a-Lab Assignment 2/EPA133a-G07-A2/model/components.py b/EPA133a-Lab_Assignment_2/EPA133a-G07-A2/model/components.py
rename from EPA133a-Lab Assignment 2/EPA133a-G07-A2/model/components.py
rename to EPA133a-Lab_Assignment_2/EPA133a-G07-A2/model/components.py
diff --git a/EPA133a-Lab Assignment 2/EPA133a-G07-A2/model/README.md b/EPA133a-Lab_Assignment_2/EPA133a-G07-A2/model/README.md
rename from EPA133a-Lab Assignment 2/EPA133a-G07-A2/model/README.md
rename to EPA133a-Lab_Assignment_2/EPA133a-G07-A2/model/README.md
diff --git a/EPA133a-Lab Assignment 2/EPA133a-G07-A2/model/model_viz.py b/EPA133a-Lab_Assignment_2/EPA133a-G07-A2/model/model_viz.py
rename from EPA133a-Lab Assignment 2/EPA133a-G07-A2/model/model_viz.py
rename to EPA133a-Lab_Assignment_2/EPA133a-G07-A2/model/model_viz.py
diff --git a/EPA133a-Lab Assignment 2/EPA133a-G07-A2/model/model_run.py b/EPA133a-Lab_Assignment_2/EPA133a-G07-A2/model/model_run.py
rename from EPA133a-Lab Assignment 2/EPA133a-G07-A2/model/model_run.py
rename to EPA133a-Lab_Assignment_2/EPA133a-G07-A2/model/model_run.py
